site/sass/components/_external-community.scss

```diff
@@ -22,4 +25,15 @@
     max-height: initial;
     width: $logo-size;
   }
+
+  &__copy-wrapper {
+    color: var(--grey-850);
+    font-family: Helvetica, Arial, sans-serif;
```

Comment:
Why is this in Helvetica instead of the font family used across the rest of the site? It'd probably be my preference to use the same font (GS Text) consistently

Reply:
Yes, I thought this was odd as well, but it is designed with Helvetica specifically. I will swap over to GS Text and we can have another look at it.

Reply:
ty